apps/bmd/test/helpers/election.ts

```diff
@@ -21,6 +22,12 @@ import {
 import { Storage } from '../../src/utils/Storage'
 
 export const election = electionSample as Election
+const electionData = JSON.stringify(election)
+export const electionDefinition = {
+  election: election,
+  electionHash: sha256(electionData),
```

Comment:
Same thing here. We don't want to `stringify` and then do the hash, we want the hash of the original JSON string.